src/python/PSetTweaks/PSetTweak.py

```diff
@@ -31,7 +37,7 @@ def __init__(self, psetName):
 #  //
 # // Assistant lambda functions
 #//
-childPSets = lambda x: [ value for value in x.__dict__.values()
+childPSets = lambda x: [ value for value in list(x.__dict__.values())
```

Comment:
shall we try to use `viewvalues` instead?

Reply:
Oops, you are definitely correct, I must have missed that. thank you very much!